### PR TITLE
Admin sendgrid stats

### PR DESCRIPTION
**Pull request by Darshan-Chauhan**
Hello Team,

This PR adds Email Stats tab in Admin panel. Which shows statastics of different event related to email. 
Ticket link: https://trello.com/c/fE0sr3oo
Please review and let me know if any changes are required.

Thanks

### CODE REVIEW

File: app/admin/email_stats.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ActiveAdmin.register_page "Email Stats" do
+  require 'sendgrid_data'
+  menu :parent => "Dashboard"
```
Comment (varunlalan) [inline app/admin/email_stats.rb]: Please use ruby 1.9 syntax.

File: app/assets/javascripts/active_admin.js [hunk 2]
```diff
@@ -7,8 +7,41 @@
...
+    $.ajax({
+      data: {"start_date" : $('#start_date').val(),
+              "end_date" : $('#end_date').val()},
+      url:$('.dates').attr('action'),
+      method:$('.dates').attr('method'),
+      dataType:'script',
+      success: function(response) {
+        $("#start_date").datepicker({ dateFormat: 'yy-mm-dd',
+          minDate:"2014-01-01", maxDate:-1});
+        $("#end_date").datepicker({ dateFormat: 'yy-mm-dd',
```
Comment (varunlalan) [inline app/assets/javascripts/active_admin.js]: We can cache start_date and end_date DOM elements and use cached variables.

File: app/helpers/active_admin/_email_stats.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+Hi
```
Comment (varunlalan) [inline app/helpers/active_admin/_email_stats.html.erb]: Is this needed?

**Comment by rohanpujaris** [comment]
@Darshan-Chauhan: Pasting conversation from Training flow here. 

> Hi @Darshan, @Manisha,
> Please check Do's and Don'ts list here  https://github.com/Kiprosh/dos_and_donts to write some quality code
> 
> RohanP
> Also check  bbatsov/ruby-style-guide: A community-driven Ruby coding style guide on GitHub

Please close this PR and follow all convention from above links and then open the PR.

File: app/helpers/active_admin/_email_stats.html.erb (hunk 3 above)
Comment (Darshan-Chauhan) [inline app/helpers/active_admin/_email_stats.html.erb]: no.. will remove it

File: app/admin/email_stats.rb (hunk 1 above)
Comment (Darshan-Chauhan) [inline app/admin/email_stats.rb]: ok

**Pull request closed without merging**